src/play-item.js

```diff
@@ -0,0 +1,42 @@
+import {setup} from './auto-advance.js';
+
+/**
+ * Removes all remote text tracks from a player.
+ *
+ * @param  {Player} player
+ */
+const clearTracks = (player) => {
+  let tracks = player.remoteTextTracks() || [];
+
```

Comment:
As in would a `forEach` actually remove all tracks?
The while loop removes the tracks from the end so that as the List gets shorter we don't end the loop early.